pepsico/projections/maproom.py

```diff
@@ -338,28 +338,31 @@ def seasonal_change(
             ac.read_data("historical", model, variable, region, unit_convert=True),
             start_month, end_month,
             start_year=start_year_ref, end_year=end_year_ref,
-        ).mean(dim="T")
+        ).mean(dim="T", keep_attrs=True)
         data = ac.seasonal_data(
             ac.read_data(scenario, model, variable, region, unit_convert=True),
             start_month, end_month,
             start_year=start_year, end_year=end_year,
-        ).mean(dim="T")
-        data = data - ref
+        ).mean(dim="T", keep_attrs=True)
+        #Tedious way to make a subtraction only to keep attributes
+        data = xr.apply_ufunc(
+            np.subtract, data.load(), ref.load(), keep_attrs="drop_conflicts"
```

Comment:
It doesn't work without the `.load()`?

Reply:
no. It returns:

  File "/home/remic/miniconda3/envs/enactsmaproom/lib/python3.9/site-packages/xarray/core/computation.py", line 692, in apply_variable_ufunc
    raise ValueError(
ValueError: apply_ufunc encountered a dask array on an argument, but handling for dask arrays has not been enabled. Either set the ``dask`` argument or load your data into memory first with ``.load()`` or ``.compute()``

Reply:
Thanks, I didn't know that. Looks like you could use `dask="allowed"` instead of multiple calls to `.load`. I don't care if you change it or not.

https://docs.xarray.dev/en/stable/generated/xarray.apply_ufunc.html

Reply:
I applied the dask option rather. Also note that there is [ufuncs.subtract](https://docs.xarray.dev/en/latest/generated/xarray.ufuncs.subtract.html) too but it dones't look like I could apply keep_attrs to that, does it?

Reply:
Resolving this and merging. Please reopen if it turns out there is something about ufuncs.subtract